src/lib.rs

```diff
@@ -28,6 +28,7 @@
 //! [Paul Williams' ANSI parser state machine]: https://vt100.net/emu/dec_ansi_parser
 #![deny(clippy::all, clippy::if_not_else, clippy::enum_glob_use)]
 #![cfg_attr(not(feature = "std"), no_std)]
+#![cfg_attr(docsrs, feature(doc_cfg))]
```

Comment:
Any reason not to use `doc_auto_cfg` instead? (https://github.com/rust-random/rand/blob/master/src/lib.rs#L56)

Reply:
Sorry, I didn't know!
https://doc.rust-lang.org/rustdoc/unstable-features.html?highlight=doc_a#doc_auto_cfg-automatically-generate-doccfg

I need to check.
Thanks.

Reply:
`#![doc(cfg_hide(doc))]` makes my head spin a bit. What a messy feature. Glad it works though.

Reply:
> Any reason not to use `doc_auto_cfg` instead? (https://github.com/rust-random/rand/blob/master/src/lib.rs#L56)

https://github.com/alacritty/vte/pull/137/commits/027150de3314b330160fa2a69f8da36855c5379c